content/en/blog/2022/jaeger-native-otlp/index.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Supporting Jaeger with native OTLP
+linkTitle: Jaeger OTLP support
+date: 2022-10-20
+author: >-
+  [Jason Plumb](https://github.com/breedx-splk) (Splunk)
+---
+
+Back in May of 2022, the Jaeger project
+[announced native support for the OpenTelemetry Protocol](https://medium.com/jaegertracing/introducing-native-support-for-opentelemetry-in-jaeger-eb661be8183c)
+(OTLP). This followed a
+[generous deprecation cycle](https://twitter.com/YuriShkuro/status/1455170693197402119)
+for the Jaeger client libraries across many languages. With these changes,
+OpenTelemetry users are now able to send traces into Jaeger with
+industry-standard OTLP, and the Jaeger client library repositories have been
+finally archived.
+
+We intend to deprecate Jaeger exporters from OpenTelemetry in the near future,
+and are looking for your feedback to determine the length of the depreation
+phase. The best way to provide feedback is by commenting on
+[the existing draft pull request](https://github.com/open-telemetry/opentelemetry-specification/pull/2858).
+
+## OpenTelemetry Support
+
+This interoperability is a wonderful victory both for Jaeger users and for
+OpenTelemetry users. However, we're not done yet. The OpenTelemetry
+specification still requires support for Jaeger client exporters across
+languages.
+
+This causes challenges for both Jaeger users and OpenTelemetry maintainers.
+
+### #1. Confusing Choices
+
+Currently, users are faced with a choice of exporter (Jaeger or OTLP), and this
+can be a source of confusion. A user might be inclined, when exporting telemetry
+to Jaeger, to simply choose the Jaeger exporter because the name matches (even
+though Jaeger now actively encourages the use of OTLP).
+
+If we can eliminate this potentially confusing choice, we can improve the user
+experience and continue standardizing on a single interoperable protocol. We
+love it when things "just work" out of the box!
+
+### #2. Maintenance and Duplication
+
+Because the Jaeger client libraries are now archived, they will not receive
+updates (including security patches). To continue properly supporting Jaeger
+client exporters, OpenTelemetry authors would be required to re-implement some
+of the functionality it had previously leveraged from the Jaeger clients.
+
+Now that Jaeger supports OTLP, this feels like a step backwards: It results in
+an increased maintenance burden with very little benefit.
+
+## Intent to Deprecate
```

Comment:
How about making it clear in the section title that feedback is wanted? That will make the request stand out:

```suggestion
## Intent to Deprecate - We'd Like Your Feedback!
```